src/main/kotlin/me/zeroeightsix/kami/feature/module/MurderMysteries.kt

```diff
@@ -0,0 +1,105 @@
+package me.zeroeightsix.kami.feature.module
+
+import io.github.fablabsmc.fablabs.api.fiber.v1.annotation.Setting
+import io.github.fablabsmc.fablabs.api.fiber.v1.annotation.Setting.Constrain
+import me.zero.alpine.listener.EventHandler
+import me.zero.alpine.listener.Listener
+import me.zeroeightsix.kami.event.TickEvent
+import me.zeroeightsix.kami.util.text
+import net.minecraft.item.*
+import net.minecraft.nbt.Tag.AQUA
+import net.minecraft.nbt.Tag.RED
+import net.minecraft.nbt.Tag.GOLD
+import net.minecraft.util.Formatting
+
+@Module.Info(
+        name = "MurderMysteries",
+        category = Module.Category.MISC,
+        description = "Get notified when a player holds a dangerous item"
+)
+object MurderMysteries : Module() {
+
+    @Setting(name = "InnocuousItems")
+    private var innocuousItems = true
+    @Setting(name = "WeaponItems")
+    private var weaponItems = true
+    @Setting(name = "ToolItems")
+    private var toolItems = true
+    @Setting(name = "DetectiveItems")
+    private var detectiveItems = false
+    @Setting(name = "IgnoreWoodenShovel", comment = "Ignore wooden shovels from ToolItems")
+    private var ignoreWoodenShovel = false
+    @Setting(name = "CooldownOnOutput", comment = "How often it will say who is murderer") // This is how often it will tell the player in chat
+    private var updateEvery: @Constrain.Range(min = 3.0, max = 16.0, step = 0.5) Float = 8f
+    @Setting(name = "AnnounceMode", comment = "Say who the murderer is in chat (WARNING: VERY EVIDENT YOU ARE USING THIS MODULE)")
+    private var announceMode = false
+    @Setting(name = "IgnoreSelf", comment = "Does not announce that you are the murderer")
+    private var ignoreSelf = true
```

Comment:
This is rather useless - obviously the player will know if they're the murderer, and announcing you're the murderer to chat is nonsensical as well. Just remove the setting.